lib/generators/templates/rpush_3_0_1_updates.rb

```diff
@@ -0,0 +1,13 @@
+class Rpush301Updates < ActiveRecord::VERSION::MAJOR >= 5 ? ActiveRecord::Migration[5.0] : ActiveRecord::Migration
+  def self.up
+    change_column_null :rpush_notifications, :mutable_content, false
+    change_column_null :rpush_notifications, :content_available, false
+    change_column_null :rapns_notifications, :alert_is_json, false
```

Comment:
I'm most likely doing something wrong (late night coding :smile: ), but this part fails for me locally (the table `rapns_notifications` doesn't exist). Shouldn't it be:
```
change_column_null :rpush_notifications, :alert_is_json, false
```

Reply:
yes!

... table got renamed :(

https://github.com/rpush/rpush/blob/2fa09cc352eae4abff8be7e0eec6324786df6e7f/lib/generators/templates/rename_rapns_to_rpush.rb

Reply:
https://github.com/rpush/rpush/pull/396